src/components/Quickstart/QuickstartPanel.astro

```diff
@@ -0,0 +1,143 @@
+---
+import { MarkdownHeading } from "astro"
+import { QuickstartsFrontmatter, productsInfo } from "~/content/config"
+import * as CONFIG from "../../config"
+import TableOfContents from "./TableOfContents"
+
+interface Props {
+  frontmatter: QuickstartsFrontmatter
+  headings: MarkdownHeading[]
+}
+const { frontmatter, headings } = Astro.props
+const currentPage = new URL(Astro.request.url).pathname
+const currentFile = `src/content${currentPage.replace(/\/$/, "")}.mdx`
```

Comment:
This might break at some point because urls might not map 1:1 to files inside the `collections` folder.

We should use the collection name and the slug to derive the correct url